bigtable-client-core-parent/bigtable-client-core/src/test/java/com/google/cloud/bigtable/grpc/TestBigtableSessionConnectionCache.java

```diff
@@ -1,145 +0,0 @@
-/*
- * Copyright 2020 Google Inc. All Rights Reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package com.google.cloud.bigtable.grpc;
-
-import com.google.api.gax.rpc.ClientContext;
-import com.google.bigtable.v2.BigtableGrpc.BigtableImplBase;
-import com.google.bigtable.v2.ReadRowsRequest;
-import com.google.bigtable.v2.ReadRowsResponse;
-import com.google.bigtable.v2.ReadRowsResponse.CellChunk;
-import com.google.cloud.bigtable.config.BigtableOptions;
-import com.google.cloud.bigtable.config.CredentialOptions;
-import com.google.cloud.bigtable.data.v2.models.Query;
-import com.google.cloud.bigtable.grpc.scanner.FlatRow;
-import com.google.protobuf.ByteString;
-import com.google.protobuf.BytesValue;
-import com.google.protobuf.StringValue;
-import io.grpc.Server;
-import io.grpc.ServerBuilder;
-import io.grpc.stub.StreamObserver;
-import java.io.IOException;
-import java.net.ServerSocket;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Map;
-import org.junit.After;
-import org.junit.Assert;
-import org.junit.Before;
-import org.junit.Test;
-
-public class TestBigtableSessionConnectionCache {
```

Comment:
why is this deleted?

Reply:
This is testing connection pool caching of veneer channels. This doesn't belong in bigtable-client-core. I need to port this to BigtableVeneerApi in bigtable-hbase. I'll revive this in a follow up PR

Reply:
Can we add it as part of this one? I don't want it getting lost in the shuffle.

Reply:
It should stand on it own. it should be ready for review shortly

Reply:
https://github.com/googleapis/java-bigtable-hbase/pull/2916